### ISSUE TITLE
Machine API doesn't support `platform: baremetal`

### ISSUE THREAD

**Issue by vrutkovs** [openshift/machine-api-operator#245]
RHHI.Next clusters fail after bootstrap is complete:

```
oc --config ocp/auth/kubeconfig -n openshift-machine-api logs machine-api-operator-5bbc744598-cs6cm | tail -n 3
E0308 20:06:07.164857       1 operator.go:176] Failed getting operator config: no platform provider found on install config
E0308 20:06:27.657099       1 operator.go:176] Failed getting operator config: no platform provider found on install config
E0308 20:07:08.622265       1 operator.go:176] Failed getting operator config: no platform provider found on install config
```
/cc @booxter

**Comment by vrutkovs** [comment]
Seem #235 would fix that

**Comment by vrutkovs** [comment]
Fixed by #246, proper implementation with operator would be added in #235
